Benchmarks/Benchmarks/P90AbsoluteThresholds/P90AbsoluteThresholds.swift

```diff
@@ -19,17 +19,18 @@ import Foundation
 #endif
 
 let benchmarks = {
-    var thresholds: [BenchmarkMetric: BenchmarkThresholds]
+    var thresholdTolerance: [BenchmarkMetric: BenchmarkThresholds]
     let relative: BenchmarkThresholds.RelativeThresholds = [.p25: 25.0, .p50: 50.0, .p75: 75.0, .p90: 100.0, .p99: 101.0, .p100: 201.0]
     let absolute: BenchmarkThresholds.AbsoluteThresholds = [.p75: 999, .p90: 1_000, .p99: 1_001, .p100: 2_001]
-    thresholds = [.mallocCountTotal: .init(relative: relative, absolute: absolute)]
+    thresholdTolerance = [.mallocCountTotal: .init(relative: relative, absolute: absolute),
+                          .syscalls: .init(relative: [.p90: 23.0], absolute: [.p90: 123])]
 
     Benchmark.defaultConfiguration = .init(metrics: [.mallocCountTotal, .syscalls] + .arc,
                                            warmupIterations: 1,
                                            scalingFactor: .kilo,
                                            maxDuration: .seconds(2),
                                            maxIterations: .kilo(100),
-                                           thresholds: thresholds)
+                                           thresholds: thresholdTolerance)
```

Comment:
Inconsistent naming, above there is a "thresholdTolerances", here it's "Tolerance".

Also I'm not sure if you could just deprecate the old initializer and update the name of that parameter in a backward-compatible way 🤔

Reply:
There is much API surface I would love to revamp (but the is alas not right now), I did that rename mostly to highlight that it is really tolerances that are defined by the source code, the actual thresholds compared against are either statically stored or gotten from a baseline/testrun.

Reply:
https://github.com/ordo-one/package-benchmark/pull/284/commits/695ad8a4c51dba826edfc9ca832f407cd8f94999